src/Directory.Build.targets

```diff
@@ -54,4 +59,39 @@
   <Target Name="DumpTargets" BeforeTargets="ResolveProjectReferences">
     <Message Text="DumpTargets> $(OutputPath), C=[$(Configuration)], CG=[$(ConfigurationGroup)], OG=[$(OSGroup)], TG=[$(TargetGroup)]" Importance="Low" />
   </Target>
+
+  <Target Name="GenerateLaunchSettingsFiles"
+          Condition="'$(LaunchSettingsOutputFileFullPath)' != '' AND '$(OmitLaunchSettings)' != 'true'"
+          Inputs="$(LaunchSettingsInputFileFullPath)"
+          Outputs="$(LaunchSettingsOutputFileFullPath)">
+
+    <PropertyGroup>
+      <LaunchSettingsFileContent>
+{
+  &quot;profiles&quot;: {
+    &quot;xUnit.net Console&quot;: {
+      &quot;commandName&quot;: &quot;Executable&quot;,
+      &quot;executablePath&quot;: &quot;$(StartProgram.Replace('\', '\\'))&quot;,
+      &quot;commandLineArgs&quot;: &quot;$(StartArguments)&quot;,
+      &quot;workingDirectory&quot;: &quot;$(StartWorkingDirectory.Replace('\', '\\'))&quot;
+    }
+  }
+}
+      </LaunchSettingsFileContent>
+    </PropertyGroup>
+
+    <MakeDir Directories="$(AppDesignerFolder)" />
+    <WriteLinesToFile File="$(LaunchSettingsOutputFileFullPath)"
+                      Lines="$(LaunchSettingsFileContent)"
+                      Overwrite="true" />
+
+  </Target>
+
+  <PropertyGroup>
```

Comment:
Can you please move this into the property group above so that it only gets added for InvokableTestProjects.